packages/fxa-shared/db/models/auth/session-token.ts

```diff
@@ -26,6 +26,25 @@ const VERIFICATION_METHOD = {
 export type VerificationMethod = keyof typeof VERIFICATION_METHOD;
 export type SessionVerifiedState = 'verified' | 'unverified';
 
+export function verificationMethodToNumber(
+  method: VerificationMethod | number
+) {
+  return typeof method === 'number' ? method : VERIFICATION_METHOD[method];
```

Comment:
What if method is a number, but isn't a valid number (ie not specified by VERIFICATION_METHOD)? Does this need to be considered?

Reply:
We'll insert it into the database, I'm making the presumption that we filter these to valid ones in the auth-server before it gets down here.

Reply:
To be more specific, I consolidated this method here, the implementation is identical to the original version.